new/NLSms1894.xml

```diff
@@ -0,0 +1,507 @@
+                                                <locus target="#4v"></locus>
+                                                <desc><term key="arch">Arch</term> resting on <term key="curtain">curtained</term> 
+                                                    <term key="column">columns</term> with three <foreign xml:lang="gez">ጢር፡</foreign> 
+                                                    <term key="bird">birds</term> and four <foreign xml:lang="gez">ከራዊያ፡</foreign> birds 
+                                                    atop, presumably intended to contain part of the Eusebian prefatory material, but 
+                                                    (originally) left empty.</desc>
+                                            </decoNote>
+                                            
+                                            <decoNote type="miniature" xml:id="d9">
+                                                <locus target="#5r"></locus>
+                                                <desc><ref type="authFile" corresp="AT1034Tempietto"/> flanked by two paradisiacal trees 
+                                                    (<foreign xml:lang="gez">ዕፀ፡ ገነት።</foreign>), two <foreign xml:lang="gez">ባቡላ፡</foreign> 
+                                                    antelopes, and two <foreign xml:lang="gez">ጕዕራ፡</foreign> <term key="bird">birds</term>. 
+                                                    There are two <foreign xml:lang="gez">ጢር፡</foreign> birds, two <foreign xml:lang="gez">ዱራ፡</foreign> 
+                                                    birds, and two <foreign xml:lang="gez">ሰገን፡</foreign> birds atop. Inside the Tempietto, the
+                                                    following inscription is found: <foreign xml:lang="gez">ኑባሬ<supplied reason="subaudible">፡</supplied> 
+                                                        ሥርዓት፡ ዘከመ፡ ኀብሩ፡ አርባዕቱ።</foreign></desc>
+                                            </decoNote>
+                                            
+                                            <decoNote type="miniature" xml:id="d10">
+                                                <locus target="#5v"></locus>
+                                                <desc><ref type="authFile" corresp="AT1005Crucifixion">Crucifixion</ref> without the Cruficied under an 
+                                                    <term key="arch">arch</term> resting on <term key="column">columns</term>. Atop the arch are 
+                                                    four <foreign xml:lang="gez">ጢር፡</foreign> <term key="bird">birds</term> and four 
+                                                    <foreign xml:lang="gez">ዱራ፡</foreign> birds. </desc>
+                                                <q n="1" xml:lang="gez">ነዋ፡ በግዑ፡ ለእግዚአብሔር፡ ዘያአ<unclear>ብ</unclear>ት፡ ኃጢአተ፡ ዓለም፡<foreign xml:lang="en">Behold the Lamb of God who takes away the sin of the world.</foreign></q>
```

Comment:
Translations of legends should follow `<q>` containing the legend in another `<q>` (see https://betamasaheft.eu/Guidelines/?id=decorationDescription). (Technically, some of these here seem to be captions, not legends)
I also don't think that @n will be visualised - did you see this used anywhere in this context?

Reply:
I followed the description of a Paris MS, but will change as you suggest :)

Reply:
Sorry, I may have been the one misleading you, I sent you the Paris record, just for showing how I had approached describing Canon Tables (which is also open to debate), This is incredibly detailed, thanks,